test/ufixed.jl

```diff
@@ -113,7 +125,7 @@ function testtrunc{T}(inc::T)
     end
 end
 
-for T in FixedPointNumbers.UF
+for T in (FixedPointNumbers.UF..., UF2...)
```

Comment:
We probably shouldn't test UF2 here since `testtrunc` tests every value under the sun and increases the testing times massively.